automation_tools/__init__.py

```diff
@@ -353,6 +353,28 @@ def setup_default_capsule(interface=None):
     )
 
 
+def setup_email_notification(smtp=None):
```

Comment:
This task seems to just configure the local email service. When the smtp has a value it will contain an URL with username, password, server and port? Maybe some doc about this parameter would make the task's docstring better.